plugins/search/src/components/SearchBar/SearchBar.tsx

```diff
@@ -40,28 +54,30 @@ export const SearchBar = ({ className, debounceTime = 0 }: Props) => {
   const handleClear = () => setValue('');
 
   return (
-    <InputBase
-      className={className}
-      data-testid="search-bar-next"
-      fullWidth
-      placeholder="Search in Backstage"
-      value={value}
-      onChange={handleQuery}
-      inputProps={{ 'aria-label': 'Search term' }}
-      startAdornment={
-        <InputAdornment position="start">
-          <IconButton aria-label="Query term" disabled>
-            <SearchIcon />
-          </IconButton>
-        </InputAdornment>
-      }
-      endAdornment={
-        <InputAdornment position="end">
-          <IconButton aria-label="Clear term" onClick={handleClear}>
-            <ClearButton />
-          </IconButton>
-        </InputAdornment>
-      }
-    />
+    <Paper className={classes.searchBar}>
```

Comment:
Will this have an undesirable double-paper effect on existing installations?

Reply:
yes, its not something we have to do now, but when I added the examples to the storybooks I just thought why ship a search bar component without the paper, buuuut its also smoother to get this merged without the change so I'll just add the paper to the storybook example instead.